### PR TITLE
Newsletters - embed - terms: Text color becomes black

### PR DESCRIPTION
**Pull request by Fatalito**
## What does this change?
Privacy wording and captcha text become black when input is focused
## Screenshots

nl-embed-dark

privacy-unfocused

### Tested

- [X] Locally

### CODE REVIEW

**Comment by Fatalito** [comment]
[Privacy notice and terms black](https://trello.com/c/hjTPYDcM/556-privacy-notice-and-terms-black)

File: common/app/views/fragments/email/signup/subscription/emailSignUp.scala.html [hunk 1]
```diff
@@ -84,11 +86,13 @@ <h2 class="email-sub__heading">@Html(emailEmbedData.title)</h2>
         document.getElementById("@inputId")
         .addEventListener("focusin", ()=> {
             document.getElementById("@privacyNoticeId")?.setAttribute('class', '@privacyNoticeVisibleClass')
+            document.getElementById("@termsId")?.setAttribute('class', 'terms @termsDark')
```
Comment (oliverlloyd) [inline common/app/views/fragments/email/signup/subscription/emailSignUp.scala.html:89]: If you use `classList.add("my-class");` then you don't need to worry about overwriting any other existing classes

https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/add

Comment (Fatalito) [reply common/app/views/fragments/email/signup/subscription/emailSignUp.scala.html]: nice

**Review by jamesgorrie — COMMENTED** [review]
Is there a reason not to use CSS `input:focus` for this?

**Comment by Fatalito** [comment]
> Is there a reason not to use CSS `input:focus` for this?

Changed to use focus-within instead of js

File: common/app/views/fragments/email/signup/subscription/emailSignUp.scala.html [hunk 2]
```diff
@@ -57,11 +53,14 @@
 
             </div>
             <button type="submit" class="email-sub__submit-button button button--tertiary button--large" id="email-embed-signup-button--old" data-component="email-signup-button @componentClass-@listName" data-link-name="@componentClass | @listName">@fragments.inlineSvg("envelope", "icon", Seq("submit-input__icon"))Sign up</button>
-            @if(EmailSignupRecaptcha.isSwitchedOn) {
+            @if(EmailSignupRecaptcha.isSwitchedOn && NewslettersRemoveConfirmationStep.isSwitchedOn ) {
+                @fragments.email.signup.recaptchaContainer()
+                @fragments.email.signup.recaptchaTerms(fragments.email.signup.privacyNoticeContent(false), "dark")
```
Comment (jamesgorrie) [inline common/app/views/fragments/email/signup/subscription/emailSignUp.scala.html:58]:
```suggestion
                @fragments.email.signup.recaptchaTerms(fragments.email.signup.privacyNoticeContent(plural = false), "dark")
```

Just for clarity

File: common/app/views/fragments/email/signup/subscription/emailSignUp.scala.html [hunk 3]
```diff
@@ -57,11 +53,14 @@
...
+            } else if(EmailSignupRecaptcha.isSwitchedOn) {
                 @fragments.email.signup.recaptchaContainer()
-                @fragments.email.signup.recaptchaTerms(fragments.email.signup.privacyNoticeContent(privacyNoticeId, privacyNoticeHiddenClass, false))
-            } else {
-                @fragments.email.signup.privacyNoticeContent(privacyNoticeId, privacyNoticeHiddenClass, false)
+                @fragments.email.signup.recaptchaTerms()
+            }else if(NewslettersRemoveConfirmationStep.isSwitchedOn) {
+                @fragments.email.signup.privacyNoticeContent(false, "terms dark")
```
Comment (jamesgorrie) [inline common/app/views/fragments/email/signup/subscription/emailSignUp.scala.html:63]:
```suggestion
                @fragments.email.signup.privacyNoticeContent(plural = false, termsClass = "terms dark")
```

**Review by jamesgorrie — APPROVED** [review]
Nitpicking - but othersise great.

File: common/app/views/fragments/email/signup/subscription/emailSignUp.scala.html (hunk 1 above)
Comment (oliverlloyd) [reply common/app/views/fragments/email/signup/subscription/emailSignUp.scala.html]: 👍

File: common/app/views/fragments/email/signup/privacyNoticeContent.scala.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
 
-@(privacyNoticeId: String = "privacy-notice", privacyNoticeClass: String  = "terms", plural: Boolean = true)(implicit request: RequestHeader)
+@(plural: Boolean = true, termsClass: String  = "")(implicit request: RequestHeader)
```
[the remark this answers is not in the document]
Comment (alinaboghiu) [reply common/app/views/fragments/email/signup/privacyNoticeContent.scala.html:2]: sorry if I'm misunderstanding this, but I think we were using it with the same privacyNoticeId in both places?

**Pull request merged**